Add Template to Core Guide to Kusama (#5879)
w3f/polkadot-wiki: docs/build/build-guides-coretime-troubleshoot.md

---
id: build-guides-coretime-troubleshoot

> title: Troubleshooting

Reviewer: ```suggestion
title: Coretime Troubleshooting FAQ
```

> sidebar_label: Troubleshooting

Reviewer: ```suggestion
sidebar_label: Troubleshooting FAQ
```

> description: Introduction to the Polkadot SDK

Reviewer: ```suggestion
description: FAQ on issues related to using coretime and testing/deploying parachains using Polkadot SDK
```

keywords: [coretime, blockspace, parathread, parachain, cores, coretime, agile]
slug: ../build-guides-coretime-troubleshoot
---

This page aims to cover and aggregate various resources that relate to troubleshooting common
problems when using the Polkadot SDK or deploying on a core.

## FAQ / Troubleshooting

### Why do I have to sync Rococo locally? Can't I just use a remote, trusted node and connect to that?

> **A:** You can actually, via the `--relay-chain-rpc-urls` flag, which can be passed to your
> collator. Unfortunately, the caveat is you can't use this for collation at this time - meaning if

Reviewer: "You can actually" sentence incomplete and needs further context. Also remove ambiguity of referring the node as a collator node, because the next sentence says the collator can't be used for collation which makes no sense. I thought you meant a Rococo node which can't be used for collation.

```suggestion
**A:** You can remotely connect to Rococo network via the --relay-chain-rpc-urls flag, which can be passed to your node. Unfortunately, the caveat is you can't use this node for collation at this time - meaning if
```

you intend on being a collator/validator for your blockchain and intend to create blocks, you need

> to sync locally.

Reviewer: ```suggestion
to sync the chain locally.
```

---

### Is there a faster way to sync Rococo? Why not warp sync?

**A:** Warp sync is currently not possible on Rococo or Westend.

> [See this answer for more context.](https://substrate.stackexchange.com/questions/9730/rococo-cant-warp-sync-stuck-at-16mb-finality-proof-download)

Reviewer: ```suggestion
[See this answer for more context.](https://substrate.stackexchange.com/questions/9730/rococo-cant-warp-sync-stuck-at-16mb-finality-proof-download). However, `--chain=rococo—-sync fast-unsafe` should provide a faster way to sync with the relay chain by skipping downloading state proofs and just verifying the block headers. 

```

---

### My collator is not producing blocks

**A:** Check these sanity checklists:

- https://substrate.stackexchange.com/questions/178/how-can-i-get-my-parachain-to-produce-blocks-sanity-checklist
- https://substrate.stackexchange.com/questions/1394/our-parachain-doesnt-produce-blocks-checklist

---

### I want to run more than one collator, how do I do that?

**A:** Ideally, you would want to run these on separate machines/servers, but you could as long as
you ensure you can provide different RPC/WebSocket and P2P ports for each collator. You also may

> need to sync a separate instance of Rococo for each collator on the same machine.

Reviewer: ```suggestion
need to sync a separate instance of Rococo for each collator on the same machine. You also will need to choose the block production mechanism like [Aura](https://substrate.stackexchange.com/questions/5572/production-like-parachain-setup-and-launch).
```

---

> ### Why do we only have one collator? Isn't it better to have more?

Reviewer: ```suggestion
### Why do we only have one collator in the parachain guides on the Wiki? Isn't it better to have more?
```

**A:** Mostly for simplicity. If we have more than one collator, we would have to also spin it up,
which would be a hassle on a single machine (it is possible though). Of course, if you had an actual
network with multiple collators, it is assumed you'd have separate VPS/servers for each.

---

### Why are we registering parathreads and not parachains?

> **A:** _Parathreads_ are a bit of an outdated term by now. They refer to what are now known as

Reviewer: ```suggestion
**A:** When registering a parachain on a relay chain, they are assigned a `ParaID`, and they are referred to as Parathreads till they start producing blocks. _Parathreads_ are a bit of an outdated term now. They refer to what are now known as
```
